From af7a1fa9c5b38660811c101537b71f3c3420eaee Mon Sep 17 00:00:00 2001
From: Rajat Soni <quic_rajson@quicinc.com>
Date: Mon, 29 May 2023 11:28:36 +0530
Subject: [PATCH] iw: Move wide band scan enum to the end

Wide band scan enum should be moved to the end as per requirement.

Signed-off-by: Rajat Soni <quic_rajson@quicinc.com>
---
 nl80211.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/nl80211.h b/nl80211.h
index 22a458f..9e0345d 100644
--- a/nl80211.h
+++ b/nl80211.h
@@ -6482,9 +6482,10 @@ enum nl80211_ext_feature_index {
 	NL80211_EXT_FEATURE_PROT_RANGE_NEGO_AND_MEASURE,
 	NL80211_EXT_FEATURE_BSS_COLOR,
 	NL80211_EXT_FEATURE_FILS_CRYPTO_OFFLOAD,
-	NL80211_EXT_FEATURE_WIDE_BAND_SCAN,
 	NL80211_EXT_FEATURE_RADAR_BACKGROUND,
 	NL80211_EXT_FEATURE_STA_MGMT_RTS_CTS,
+	NL80211_EXT_FEATURE_WIDE_BAND_SCAN =
+				NL80211_EXT_FEATURE_STA_MGMT_RTS_CTS + 3,
 
 	/* add new features before the definition below */
 	NUM_NL80211_EXT_FEATURES,
-- 
2.17.1

